Userland/Applications/PixelPaint/Tools/TextTool.cpp

```diff
@@ -0,0 +1,286 @@
+/*
+ * Copyright (c) 2022, Timothy Slater <tslater2006@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include "TextTool.h"
+#include "../ImageEditor.h"
+#include "../Layer.h"
+#include "LibGUI/FontPicker.h"
+#include <LibGUI/Action.h>
+#include <LibGUI/BoxLayout.h>
+#include <LibGUI/Button.h>
+#include <LibGUI/Label.h>
+#include <LibGUI/Menu.h>
+#include <LibGUI/Painter.h>
+#include <LibGUI/TextBox.h>
+#include <LibGfx/Bitmap.h>
+#include <LibGfx/Palette.h>
+namespace PixelPaint {
```

Comment:
Nit-pick: A blank line between the includes and the namespace please.

Reply:
Fixed